File: FCUBEAPI/FleetMasters/Models/TyreModelMasterList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetMasters.Models
{
    public class TyreModelMasterList
    {
        public List<TyreModelMasterModel> TyreList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }

    }
}

File: FCUBEAPP/src/app/models/tyremodelmasterlist.ts
import { Paginationmodel } from "./paginationmodel";
import { Tyremodelmastermodel } from "./tyremodelmastermodel";

export class Tyremodelmasterlistmodel {
    tyreList: Tyremodelmastermodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}